fs/config/config.go

```diff
@@ -246,9 +246,11 @@ func makeConfigPath() string {
 	// variable, since then we skip actually trying to create the default
 	// and report any errors related to it (we can't use pflag for this because
 	// it isn't initialised yet so we search the command line manually).
+	// Use processed command line from fs.Args so we can pick `-o config=...`
+	// if rclone was invoked as a mount helper.
 	_, configSupplied := os.LookupEnv("RCLONE_CONFIG")
 	if !configSupplied {
-		for _, item := range os.Args {
+		for _, item := range fs.Args {
 			if item == "--config" || strings.HasPrefix(item, "--config=") {
```

Comment:
This `fs/config` code can be used very early so we run conversion super early, in the `init` block of `fs` package.